minard/noisedb.py

```diff
@@ -35,6 +35,7 @@ def runs_after_run(run, maxrun = '+inf'):
     Requires Redis instance, start-run and maximum run.
     '''
     keys = redis.zrangebyscore(RUN_INDEX, run, maxrun)
+    keys = keys.reverse() # Sort with most recent first
```

Comment:
reverse does an in place reverse. You can either do `keys = reversed(keys)` or `keys.reverse()`.